src/constants.ts

```diff
@@ -31,6 +33,8 @@ export const isEndGroup = (t: string): boolean => /##\[endgroup\]/.test(t)
 export const githubAuthProviderId: string = 'github';
 export const githubScopes: string[] = ['repo', 'workflow', 'admin:public_key'];
 
+export const templateReposUsername: string = 'staticwebdev';
```

Comment:
I think this is only used in the one spot, so we probably don't need to throw this in the constants file.  We don't want to unnecessarily clutter this file up.